query4-9.sql

```diff
@@ -55,23 +55,23 @@ WHERE SHOP_REVENUE.Total_revenue = (select MAX(Total_revenue) from SHOP_REVENUE)
 -- query 7
 -- Find the user that made the most amount of complaints
 WITH COMPLAINT_COUNT AS (
-      SELECT UID
+      SELECT UID, COUNT(*) as Complaint_num
       FROM COMPLAINTS
-      GROUP BY UID
-      HAVING COUNT(*) >= ALL (SELECT COUNT(*)
-                              FROM COMPLAINTS
-                              GROUP BY UID)),
+      GROUP BY UID),
+USER_WITH_MOST AS (
+      SELECT UID
+      FROM COMPLAINT_COUNT
+      WHERE Complaint_num = (SELECT MAX(Complaint_num) FROM COMPLAINT_COUNT)),
```

Comment:
I think these two ways are equivalent and both are ok. I learned this method in the tutorial last week.

Reply:
I mean the >= ALL method